src/polus/plugins/_plugins/classes/plugin_classes.py

```diff
@@ -380,13 +378,40 @@ def submit_plugin(
 
     # Save the manifest if it doesn't already exist in the database
     organization = plugin.containerId.split("/")[0]
-    org_path = PLUGIN_DIR.joinpath(organization.lower())
+    org_path = _PLUGIN_DIR.joinpath(organization.lower())
     org_path.mkdir(exist_ok=True, parents=True)
     if not org_path.joinpath(out_name).exists():
-        with open(org_path.joinpath(out_name), "w") as fw:
-            json.dump(plugin.dict(), fw, indent=4)
+        with open(org_path.joinpath(out_name), "w", encoding="utf-8") as file:
+            manifest_ = plugin.dict()
+            manifest_["version"] = manifest_["version"]["version"]
```

Comment:
I think I so in a later PR that you intended to do it, but it seems to be already in this one.

Reply:
Different functionality